### PR TITLE
Integration Tests

### PR DESCRIPTION
**Pull request by mambax**
Trying to reproduce https://gitlab.com/mambax/exman2/-/merge_requests/4/diffs.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM openjdk:16-jdk-alpine
 RUN addgroup -S spring && adduser -S spring -G spring
 USER spring:spring
-ARG JAR_FILE=build/libs/*.jar
+ARG JAR_FILE=build/libs/*T.jar
```
Comment (mambax) [inline Dockerfile:4]: This is ugly how does one do that?

File: build.gradle [hunk 2]
```diff
@@ -17,11 +18,30 @@ dependencies {
...
+	useJUnitPlatform{
+	    includeTags 'unit'
+	}
+    testLogging {
+        events "passed", "skipped", "failed"
+    }
 }
+
+task integrationTest(type: Test) {
+    dependsOn build
```
Comment (mambax) [inline build.gradle:36]: this i do not like, but it breaks otherwise when build dir is clean :S

**Comment by mambax** [comment]
Hmm, either the two tasks used the same runner or somethings odd because it shows I have 5 integration tests but I only have 1.

**Comment by mambax** [comment]
Okay, confirmed that for integration test he runs all 5... Why? Does include not explicitly exclude?

File: build.gradle (hunk 2 above)
Comment (pazone) [reply build.gradle:36]: it's fine. Gradle won't build it from scratch if it already built. It will use cache

**Pull request merged**